release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -650,6 +650,18 @@ If you disable the `ImageRegistry` capability or if you disable the integrated {
 
 For more information, see xref:../nodes/pods/nodes-pods-secrets.adoc#auto-generated-sa-token-secrets_nodes-pods-secrets[Automatically generated secrets].
 
+[discrete]
+[id="ocp-4-15-ovnic-default-range"]
+==== Open Virtual Network Infrastructure Controller default range
+
+Previously, the IP range `168.254.0.0/16` was the default IP used for the transit switch subnet in the Open Virtual Network Infrastructure Controller. With this update, the `100.88.0.0/16` IP range is now the default IP used. Customers should not use this IP range in their infrastructure network. (link:https://issues.redhat.com/browse/OCPBUGS-20178[*OCPBUGS-20178*])	
+
+[discrete]
+[id="egress-ip-azure-private-cluster"]
+==== Application of egress IP addresses to egress nodes on private Azure clusters
+
+In Microsoft Azure setups, there's a limitation where secondary IPs designated as egress IPs lack outbound connectivity. This means that pods associated with these IPs are unable to access the internet. However, they can still reach external servers within the infrastructure network, which is the intended use case for egress IPs. This update enables egress IPs for Microsoft Azure setups, allowing outbound connectivity to be achieved through outbound rules. (link:https://issues.redhat.com/browse/OCPBUGS-5491[*OCPBUGS-5491*])	
```

Comment:
I would suggest: "Microsoft Azure clusters have a limitation where..."

Also consider replacing "IPs" with "IP addresses" everywhere in the files.